pkg/api/validate.go

```diff
@@ -454,6 +454,9 @@ func (v *Validator) validateNetworkProfile(np *NetworkProfile) (errs []error) {
 		errs = append(errs, fmt.Errorf("networkProfile cannot be nil"))
 		return
 	}
+	if np.VnetID != "" && !rxVNetID.MatchString(np.VnetID) {
+		errs = append(errs, fmt.Errorf("invalid properties.networkProfile.VnetId %q", np.VnetID))
```

Comment:
the rest of the error messages start the field lowercased so this should be "invalid properties.networkProfile.vnetId %q"

Reply:
fixed, changed to camelCase